File: app/src/main/java/vn/loitp/app/activity/api/retrofit2/SOService.java
package vn.loitp.app.activity.api.retrofit2;

import retrofit2.Call;
import retrofit2.http.GET;

public interface SOService {

    @GET("/answers?order=desc&sort=activity&site=stackoverflow")
    Call<SOAnswersResponse> getAnswers();

    //@GET("/answers?order=desc&sort=activity&site=stackoverflow")
    //Call<SOAnswersResponse> getAnswers(@Query("tagged") String tags);
}

File: app/src/main/java/vn/loitp/up/a/api/retrofit2/SOService.kt
package vn.loitp.up.a.api.retrofit2

import retrofit2.Call
import retrofit2.http.GET
import vn.loitp.up.a.api.retrofit2.md.SOAnswersResponse

interface SOService {
    @get:GET("/answers?order=desc&sort=activity&site=stackoverflow")
    val answers: Call<SOAnswersResponse>
}
